src/cpp/rtps/transport/shared_mem/SharedMemGlobal.hpp

```diff
@@ -70,7 +70,7 @@ class SharedMemGlobal
     typedef MultiProducerConsumerRingBuffer<BufferDescriptor>::Listener Listener;
     typedef MultiProducerConsumerRingBuffer<BufferDescriptor>::Cell PortCell;
 
-    static const uint32_t CURRENT_ABI_VERSION = 3;
+    static const uint32_t CURRENT_ABI_VERSION = 5;
```

Comment:
OK, this is because changes in different commits. I do not clean shm/dev. Let's left it this way, for safety,  in this PR and I will do well for the next ones.